{
    "containers": {
        "cna": {
            "affected": [
                {
                    "product": "n/a",
                    "vendor": "n/a",
                    "versions": [
                        {
                            "status": "affected",
                            "version": "n/a"
                        }
                    ]
                }
            ],
            "datePublic": "2015-09-30T00:00:00",
            "descriptions": [
                {
                    "lang": "en",
                    "value": "Node.js 4.0.0, 4.1.0, and 4.1.1 allows remote attackers to cause a denial of service."
                }
            ],
            "problemTypes": [
                {
                    "descriptions": [
                        {
                            "description": "n/a",
                            "lang": "en",
                            "type": "text"
                        }
                    ]
                }
            ],
            "providerMetadata": {
                "dateUpdated": "2017-10-14T09:57:01",
                "orgId": "8254265b-2729-46b6-b9e3-3dfca2d5bfca",
                "shortName": "mitre"
            },
            "references": [
                {
                    "tags": [
                        "x_refsource_CONFIRM"
                    ],
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1268791"
                },
                {
                    "tags": [
                        "x_refsource_CONFIRM"
                    ],
                    "url": "https://github.com/nodejs/node/issues/3138"
                },
                {
                    "name": "101260",
                    "tags": [
                        "vdb-entry",
                        "x_refsource_BID"
                    ],
                    "url": "http://www.securityfocus.com/bid/101260"
                }
            ],
            "x_legacyV4Record": {
                "CVE_data_meta": {
                    "ASSIGNER": "cve@mitre.org",
                    "ID": "CVE-2015-7384",
                    "STATE": "PUBLIC"
                },
                "affects": {
                    "vendor": {
                        "vendor_data": [
                            {
                                "product": {
                                    "product_data": [
                                        {
                                            "product_name": "n/a",
                                            "version": {
                                                "version_data": [
                                                    {
                                                        "version_value": "n/a"
                                                    }
                                                ]
                                            }
                                        }
                                    ]
                                },
                                "vendor_name": "n/a"
                            }
                        ]
                    }
                },
                "data_format": "MITRE",
                "data_type": "CVE",
                "data_version": "4.0",
                "description": {
                    "description_data": [
                        {
                            "lang": "eng",
                            "value": "Node.js 4.0.0, 4.1.0, and 4.1.1 allows remote attackers to cause a denial of service."
                        }
                    ]
                },
                "problemtype": {
                    "problemtype_data": [
                        {
                            "description": [
                                {
                                    "lang": "eng",
                                    "value": "n/a"
                                }
                            ]
                        }
                    ]
                },
                "references": {
                    "reference_data": [
                        {
                            "name": "https://bugzilla.redhat.com/show_bug.cgi?id=1268791",
                            "refsource": "CONFIRM",
                            "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1268791"
                        },
                        {
                            "name": "https://github.com/nodejs/node/issues/3138",
                            "refsource": "CONFIRM",
                            "url": "https://github.com/nodejs/node/issues/3138"
                        },
                        {
                            "name": "101260",
                            "refsource": "BID",
                            "url": "http://www.securityfocus.com/bid/101260"
                        }
                    ]
                }
            }
        },
        "adp": [
            {
                "providerMetadata": {
                    "orgId": "96f1cf3c-f80a-4e0f-a063-99a91e3f1325",
                    "shortName": "secretariat-reference",
                    "dateUpdated": "2024-06-30T17:30:19.927Z"
                },
                "references": [
                    {
                        "tags": [
                            "x_refsource_CONFIRM",
                            "x_cve_prog_secretariat_snapshot"
                        ],
                        "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1268791/DEMO_2024_06_19"
                    },
                    {
                        "tags": [
                            "x_refsource_CONFIRM",
                            "x_cve_prog_secretariat_snapshot"
                        ],
                        "url": "https://github.com/nodejs/node/issues/3138/DEMO_2024_06_19"
                    },
                    {
                        "name": "101260",
                        "tags": [
                            "vdb-entry",
                            "x_refsource_BID",
                            "x_cve_prog_secretariat_snapshot"
                        ],
                        "url": "http://www.securityfocus.com/bid/101260/DEMO_2024_06_19"
                    },
                    {
                        "tags": [
                            "x_refsource_CONFIRM",
                            "x_cve_prog_secretariat_snapshot",
                            "x_cve_prog_secretariat_snapshot@2024-06-28T20:08:51.933Z"
                        ],
                        "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1268791/adp-test-6-28"
                    },
                    {
                        "tags": [
                            "x_refsource_CONFIRM",
                            "x_cve_prog_secretariat_snapshot",
                            "x_cve_prog_secretariat_snapshot@2024-06-28T20:08:51.933Z"
                        ],
                        "url": "https://github.com/nodejs/node/issues/3138/adp-test-6-28"
                    },
                    {
                        "name": "101260",
                        "tags": [
                            "vdb-entry",
                            "x_refsource_BID",
                            "x_cve_prog_secretariat_snapshot",
                            "x_cve_prog_secretariat_snapshot@2024-06-28T20:08:51.933Z"
                        ],
                        "url": "http://www.securityfocus.com/bid/101260/adp-test-6-28"
                    }
                ]
            }
        ]
    },
    "cveMetadata": {
        "assignerOrgId": "8254265b-2729-46b6-b9e3-3dfca2d5bfca",
        "assignerShortName": "mitre",
        "cveId": "CVE-2015-7384",
        "datePublished": "2017-10-10T16:00:00",
        "dateReserved": "2015-09-28T00:00:00",
        "dateUpdated": "2024-06-30T17:30:19.927Z",
        "state": "PUBLISHED"
    },
    "dataType": "CVE_RECORD",
    "dataVersion": "5.1"
}